plugins/opentelemetry/wiring.go

```diff
@@ -1,25 +1,68 @@
-// Package opentelemetry provides two plugins:
-// (i)  a plugin to generate and include an opentelemetry collector instance in a Blueprint application
-// (ii) provides a modifier plugin to wrap the service with an OpenTelemetry wrapper to generate OT compatible traces/logs.
+// Package opentelemetry provides three plugins:
 //
-// The package provides an in-memory trace exporter implementation and a go-client for generating traces on both the server and client side.
-// The generated clients handle context propagation correctly on both the server and client sides.
+//  1. a plugin to wrap the service with an OpenTelemetry wrapper to generate OT compatible traces by starting and stopping client spans for remote calls between services and correctly propagating context between services. The traces are then exported to a collector client such as jaeger or zipkin.
```

Comment:
> A plugin to instrument services so that RPC calls will create opentelemetry spans and propagate opentelemetry contexts.  The plugin does this by creating wrapper classes for the service's client and server.  Traces are exported to a trace collector such as [jaeger] or [zipkin]

In the above, add links to the [jaeger] and [zipkin] plugins.